### DISCUSSION TITLE
FreeBSD Install Recommendations

### DISCUSSION THREAD

**Discussion by BugHunter1000** [Eloston/ungoogled-chromium#1361] [Q&A]
Hi.  Thanks for your time.

What is the best route to get UC running on FreeBSD?  I accept it may not be officially supported, but I want to give it a go.  Willing to patch in Flatpak but it looks also unsupported, so any advice is very appreciated.  Thx.

**Comment by wchen342** [comment]
There is a [third party script](https://github.com/Martinfx/ungoogled-chromium-freebsd/blob/main/build2.sh), but note that I cannot confirm the authenticity of it.
